### PR TITLE
Updates README to better reflect new Binauth changes

### PR DESCRIPTION
**Pull request by anderssonw**
Only parts of the readme were updated in #46, this sorts out a few of those missing changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +182,7 @@ Additionally, if image_url is not supplied neither Trivy nor Binary Attestation
 - Runs Trivy, a comprehensive security scanner. Does not run on draft pull requests.
 - Creates a binary attestation on the supplied image if trivy is run.
 - Calls the Github Security Code Scanning API and fails with exit code 1 if there are any _high_ or _critical_ errors.
-- Creates a binary attestation on the supplied image if both TFsec and Trivy scans are run and there are no _high_ or _critical_ errors in Github Security Code Scanning.
+- The above step can be overriden using the `allow_severity_level` input.
```
Comment (lislei) [inline README.md:185]:
```suggestion
- The above step can be configured by specifying the `allow_severity_level` parameter.
```

Jeg mener "overridden" gir litt andre assosiasjoner. Dette steget er mandatory da tilstedeværelsen av en slik attestering er påkrevet :)

Comment (anderssonw) [reply README.md:185]: Fair, but GitHub uses "input" as the name for things you send into GitHub workflows, so I'd prefer the use of that word :)

File: README.md [hunk 2]
```diff
@@ -372,8 +374,6 @@ jobs:
       auth_project_number: "123456789123"
       service_account: sa-name@project-dev-123.iam.gserviceaccount.com
       image_url: ${{ needs.build.outputs.image_tag_url}} # optional, must have format <registry>/<repository>:<tag>
```
Comment (lislei) [inline README.md:376]: Documentation of the input parameter `image_url` is not consistent. 

The main issue IMHO is that REAMDE.md duplicates this description. It is well documented in the source file [run-security-scans.yml]([.github](.github/workflows)/run-security-scans.yml)

File: README.md [hunk 3]
```diff
@@ -180,11 +182,11 @@ Additionally, if image_url is not supplied neither Trivy nor Binary Attestation
 - Runs Trivy, a comprehensive security scanner. Does not run on draft pull requests.
 - Creates a binary attestation on the supplied image if trivy is run.
```
Comment (lislei) [inline README.md:183]:
```suggestion
- Creates a binary attestation on the supplied image if Trivy is run.
```

File: README.md [hunk 4]
```diff
@@ -180,11 +182,11 @@ Additionally, if image_url is not supplied neither Trivy nor Binary Attestation
...
 - Calls the Github Security Code Scanning API and fails with exit code 1 if there are any _high_ or _critical_ errors.
```
Comment (lislei) [inline README.md:184]:
```suggestion
- Calls the GitHub Security Code Scanning API and fails workflow if there are any _high_ or _critical_ errors.
```

File: README.md (hunk 1 above)
Comment (lislei) [reply README.md:185]: Smooth - i changed it to "input parameter". I believe most devs will be familiar with the term:)

**Review by lislei — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -217,8 +219,8 @@ jobs:
       auth_project_number: "123456789123"
       service_account: sa-name@project-dev-123.iam.gserviceaccount.com
       image_url: <registry>/<repository>:<tag> or <registry>/<repository>@<digest> # the image created by build job
-      trivy: <optional>
-      tfsec: <optional>
+      trivy: true # default true, optional
+      tfsec: true # default true, optional
       allow_severity_level: medium
```
Comment (lislei) [inline README.md:224]: With a security perspective I'd like to argue that a better example would be to tighten the security and not weaken it. Especially for templates and cut and paste examples! :)

**Review by lislei — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (lislei) [reply README.md:224]: nvm, i had that completely wrong

File: README.md (hunk 2 above)
Comment (anderssonw) [reply README.md:376]: is the inconsistency whether or not you may use a digest directly or not? not sure i follow

Comment (lislei) [reply README.md:376]: Yes, that's exactly what I had in mind. 
Now realizing this is part of an working example, the inline comment might be a bit misleading here. 
Perhaps it should be `# the image created by build job` or even better removed all together?

Comment (anderssonw) [reply README.md:376]: I don't mind either way :)

Comment (lislei) [reply README.md:376]: Id say its already well documented, twice actually, and less is more :)

**Review by lislei — APPROVED** [review]

**Review by berg-thom — APPROVED** [review]

**Pull request merged**